SQL-Query/Query-X./SQLAnswer.md

```diff
@@ -0,0 +1,11 @@
+**10. Orders brokered but not shipped:**
+```sql
+select 
+  ORDER_ID 
+from 
+  order_header 
+where 
+  ORIGIN_FACILITY_ID != '_NA_' 
```

Comment:
It should be facility id from order item ship group and not the origin facility id from Order Header.